app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] According to the requirement, it's expected that an email confirmation feature be implemented so one is sent a confirmation link when signed up. Kindly enable comfortable so email confirmation works as expected 
- [ ] Also, I tried resetting the password but I couldn't. Please work on that as well 👍🏽

```ruby
devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable
```

Reply:
Thanks for your feedback